hooks/__tests__/useDownloadLink.test.ts

```diff
@@ -27,14 +27,12 @@ describe('useDetectOS', () => {
       writable: true,
     });
 
-    const { result } = renderHook(() => useDetectOS());
+    const { result } = renderHook(() =>
+      useDownloadLink({ version: 'v18.16.0' })
+    );
 
     await waitFor(() => {
-      expect(result.current.userOS).toBe('WIN');
-    });
-
-    await waitFor(() => {
-      expect(result.current.getDownloadLink('v18.16.0')).toBe(
+      expect(result.current).toBe(
```

Comment:
Since we don't return a function but a static value now, the reference won't change on every render unless the value actually changes, which will cause less re-render issues.